pkg/acsengine/defaults.go

```diff
@@ -496,6 +497,10 @@ func setMasterProfileDefaults(a *api.Properties, isUpgrade bool) {
 			a.MasterProfile.Distro = api.AKS
 		}
 	}
+	// set default to VMAS for now
+	if len(a.MasterProfile.AvailabilityProfile) == 0 {
```

Comment:
Does this need to be similar to the defaults logic for AgentPoolProfiles? In agentProfiles, VMSS is default availability profile for k8s versions 1.10.2 or higher. Other versions have AvailabilitySets as their default.

Reply:
Got it!